src/AppFrame.tsx

```diff
@@ -7,10 +7,23 @@ import { TwitterTimelineEmbed } from "react-twitter-embed";
 
 export function AppFrame() {
   const location = useLocation();
+  const [isMobileNavigationActive, setIsMobileNavigationActive] = useState(
+    false,
+  );
+
+  const handleMobileNavigationToggle = useCallback(
```

Comment:
Why are you memoizing the function?

Reply:
To keep referential equality when passing it as a prop to `TopBar`. It's most likely trivial here but can help with render performance in some situations.

Reply:
It can also make performance worse: https://kentcdodds.com/blog/usememo-and-usecallback

Unless you are actually seeing poor perf, avoid trying to optimize prematurely.